labs-ios-starter/CityDashboardViewController.swift

```diff
@@ -22,9 +22,14 @@ class CityDashboardViewController: UIViewController, UICollectionViewDataSource
     var cityProperties: [Int?] {
         return [city?.livability, city?.walkability, city?.traffic, city?.pollution, city?.crime, city?.rentalPrice]
     }
-    
+    var controller: ApiController?
     
     // MARK: - Actions
+    
+    @IBAction func backButtonPressed(_ sender: Any) {
+        dismiss(animated: true, completion: nil)
+    }
```

Comment:
This seems a bit odd. Back button is usually used to pop a view controller off of a navigation stack, but this looks like it may be dismissing a modal. If that is the case, I would rename this IBAction and the button with `cancel` instead of back.